06_loops_if_functions.ipynb

```diff
@@ -4,20 +4,21 @@
    "cell_type": "markdown",
```

Comment:
<p>Sounds like a formative assessment. Is it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>